src/main/kotlin/ch/wisv/choice/exam/service/ExamServiceImpl.kt

```diff
@@ -38,7 +38,7 @@ class ExamServiceImpl(val examRepository: ExamRepository, val courseService: Cou
     }
 
     override fun getExamById(id: Long): Exam
-        = examRepository.findOne(id) ?: throw CHoiceException("Exam with id $id not found")
+        = examRepository.findById(id).orElse(null) ?: throw CHoiceException("Exam with id $id not found")
```

Comment:
Hier ook de vraag of deze wijziging nodig is?

Reply:
Is dit te schalen onder legacy fix?